### ISSUE TITLE
Problem creating a new project

### ISSUE THREAD

**Issue by homerokzam** [DevExpress/devextreme-cli#312]
# Bug Report

Problem creating a new project with the command:
npx -p devextreme-cli devextreme new react-app app-name

ERROR:

/libsass/src/units.o Release/obj.target/libsass/src/libsass/src/utf8_string.o Release/obj.target/libsass/src/libsass/src/util.o Release/obj.target/libsass/src/libsass/src/values.o
  c++ '-DNODE_GYP_MODULE_NAME=binding' '-DUSING_UV_SHARED=1' '-DUSING_V8_SHARED=1' '-DV8_DEPRECATION_WARNINGS=1' '-DV8_DEPRECATION_WARNINGS' '-DV8_IMMINENT_DEPRECATION_WARNINGS' '-D_GLIBCXX_USE_CXX11_ABI=1' '-D_DARWIN_USE_64_BIT_INODE=1' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-DBUILDING_NODE_EXTENSION' -I/Users/homerokzam/.node-gyp/16.2.0/include/node -I/Users/homerokzam/.node-gyp/16.2.0/src -I/Users/homerokzam/.node-gyp/16.2.0/deps/openssl/config -I/Users/homerokzam/.node-gyp/16.2.0/deps/openssl/openssl/include -I/Users/homerokzam/.node-gyp/16.2.0/deps/uv/include -I/Users/homerokzam/.node-gyp/16.2.0/deps/zlib -I/Users/homerokzam/.node-gyp/16.2.0/deps/v8/include -I../../nan -I../src/libsass/include  -O3 -gdwarf-2 -mmacosx-version-min=10.7 -arch x86_64 -Wall -Wendif-labels -W -Wno-unused-parameter -std=c++11 -stdlib=libc++ -fno-rtti -fno-exceptions -fno-strict-aliasing -MMD -MF ./Release/.deps/Release/obj.target/binding/src/binding.o.d.raw   -c -o Release/obj.target/binding/src/binding.o ../src/binding.cpp
In file included from ../src/binding.cpp:1:
In file included from ../../nan/nan.h:56:
In file included from /Users/homerokzam/.node-gyp/16.2.0/include/node/node.h:63:
In file included from /Users/homerokzam/.node-gyp/16.2.0/include/node/v8.h:30:
/Users/homerokzam/.node-gyp/16.2.0/include/node/v8-internal.h:452:38: error: no template named 'remove_cv_t' in namespace 'std'; did you mean 'remove_cv'?
            !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
                                ~~~~~^~~~~~~~~~~
                                     remove_cv
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1/type_traits:776:50: note: 'remove_cv' declared here
template <class _Tp> struct _LIBCPP_TEMPLATE_VIS remove_cv
                                                 ^
1 error generated.
make: *** [Release/obj.target/binding/src/binding.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/homerokzam/Projects/React/app-name/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (node:events:365:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
gyp ERR! System Darwin 20.5.0
gyp ERR! command "/usr/local/Cellar/node/16.2.0/bin/node" "/Users/homerokzam/Projects/React/app-name/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /Users/homerokzam/Projects/React/app-name/node_modules/node-sass
gyp ERR! node -v v16.2.0
gyp ERR! node-gyp -v v3.8.0




**Package version:**

    devextreme-cli version: lastest

**Comment by OlegKipchatov** [comment]
Hi @homerokzam
Thanks for paying attention to this issue
I have reproduced the issue locally and we will research why it occurs. At the moment, as an immediate solution, change your node version from 16 to 14.

**Comment by babich-a** [comment]
fixed fo react app in 1.2.16
